content/en/docs/contribute/review/for-approvers.md

```diff
@@ -190,7 +190,7 @@ close the issue without fixing.
 
 Some docs issues are actually issues with the underlying code, or requests for
 assistance when something, for example a tutorial, doesn't work.
-For issues unrelated to docs, close the issue with the `triage/support` label and a comment
+For issues unrelated to docs, close the issue with the `kind/support` label and a comment
```

Comment:
👋  @mrbobbytables . A couple of questions:

- What follows when an issue is closed with this label applied?
- Where can I find more information about this label (prow)? Will this label appear in the list of labels available from the site?

Reply:
Essentially its the same label, it's just been "moved" to `kind/*` as it better reflects the type of thing it is.^^;;;